fe1-web/src/features/witness/network/messages/WitnessRegistry.ts

```diff
@@ -0,0 +1,78 @@
+import { ActionType, MessageData, ObjectType } from 'core/network/jsonrpc/messages';
```

Comment:
why duplicate this and not add a field in `MessageRegistry` ?

Reply:
My thought was that it should be inside the feature and not part of the core?

Reply:
I agree that it's part of the feature, but on the other side it could easily fit in the `MessageRegistry`, as all features depend on this too. Maybe you can talk about it with @0xdieman.

Reply:
👍 I will do that! :)